src/redux/reducers/app/index.js

```diff
@@ -35,11 +21,11 @@ export default function (state = initialState, action) {
         case DONE_APP_INITIALIZATION:
             return state.set('isInitializing', false);
 
-        case START_APP_LOADING:
-            return startAppLoading(state, action.title, action.message);
+        case SHOW_ERROR_MESSAGE:
+            return state.set('errorMessage', action.error.message);
```

Comment:
Let's try to not set state directly here instead use a function like startAppLoading.
Also in the action you say:
```
return {
        type: SHOW_ERROR_MESSAGE,
        message: error.message ? error.message : String(error),
}
```
then here must be:
```state.set('errorMessage', action.message);```
and not ```action.error.message```